modules/porous_flow/test/src/vectorpostprocessors/DiracPointsWriter.C

```diff
@@ -0,0 +1,44 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "DiracPointsWriter.h"
+
+registerMooseObject("PorousFlowTestApp", DiracPointsWriter);
+
+InputParameters
+DiracPointsWriter::validParams()
+{
+  return GeneralVectorPostprocessor::validParams();
+}
+
+DiracPointsWriter::DiracPointsWriter(const InputParameters & parameters)
+  : GeneralVectorPostprocessor(parameters),
+    _xs(declareVector("x")),
+    _ys(declareVector("y")),
+    _zs(declareVector("z"))
+{
+}
+
+void
+DiracPointsWriter::execute()
+{
+  _xs.clear();
+  _ys.clear();
+  _zs.clear();
+  std::set<Point> points;
```

Comment:
`std::set<Point>` for sorting?

Reply:
Yes - the sorting matters for testing purposes - otherwise depending on the platform, you could get the points outputted in an inconsistent order - which actually caused test failures initially.